.editorconfig

```diff
@@ -0,0 +1,40 @@
+## EditorConfig is awesome: http://EditorConfig.org
+root = true
+
+
+## Newlines section
+
+# Make sure all files use lf as line ending
+[*]
+end_of_line = lf
+
+# Make sure our have an empty line at the end
+[*.{coffee,htaccess,html.twig,js,md,php}]
+insert_final_newline = true
+
+## Charset section
+[*.{coffee,htaccess,js,md,php,twig,xml,yml}]
+charset = utf-8
+
+
+## Indenting section
+
+# 4 spaces indenting
+[*.{json,php,xml,yml}]
+indent_size = 4
+indent_style = space
+
+# 2 spaces indenting
+[*.{coffee,css,html.twig,rb,scs}]
```

Comment:
I'd also add 'html' without twig. You may have plain old html files in your project.